src/providers/krb5/krb5_child.c

```diff
@@ -2492,13 +2492,6 @@ static errno_t kuserok_child(struct krb5_req *kr)
 
     DEBUG(SSSDBG_TRACE_LIBS, "Verifying if principal can log in as user\n");
 
-    /* krb5_kuserok tries to verify that kr->pd->user is a locally known
-     * account, so we have to unset _SSS_LOOPS to make getpwnam() work. */
-    if (unsetenv("_SSS_LOOPS") != 0) {
-        DEBUG(SSSDBG_CRIT_FAILURE, "Failed to unset _SSS_LOOPS, "
-                  "krb5_kuserok will most certainly fail.\n");
-    }
-
```

Comment:
Why is this removed?

`krb5_child` is executed by `sssd_be` via fork+exec, i.e. env is copied. But `sssd_be` has this var set to 'NO' and according to the comment this is not what we want.

Reply:
You are right. `sssd_be` should set `_SSS_LOOPS=NO` and these deleted lines should not be required.